api/core/rag/datasource/vdb/tidb_vector/tidb_vector.py

```diff
@@ -104,14 +105,14 @@ def _create_collection(self, dimension: int):
                         text TEXT NOT NULL,
```

Comment:
I'm not sure, is it possible to use Int type as the primary key? If this is possible then it can achieve better performance.

Reply:
The id column in uuid is designed as aligned with other vdb types.

Reply:
Got it, thanks for the background!